webrtc.html

```diff
@@ -3295,6 +3295,10 @@
         <var>connection</var>, run the following steps:</p>
         <ol>
           <li>
+            <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+            <code>true</code>, abort these steps.</p>
+          </li>
+          <li>
```

Comment:
This is great. Unfortunately, there's also this doozie of an exception in step 2.7 of the [set description algorithm](http://w3c.github.io/webrtc-pc/#set-description):

*"If connection's signaling state is now stable, update the negotiation-needed flag. If connection's [[ needNegotiation]] slot was true both before and after this update, **queue a task to check connection's [[needNegotiation]] slot and, if still true, fire a simple event named negotiationneeded at connection**."*